### ISSUE TITLE
 这个项目时缺少一个loss_funcs.py文件吗？

### ISSUE THREAD

**Issue by AgouziY** [nbdszw/CSTnet#1]
![e9e743c226898079a27a86fdcbcacdbb](https://github.com/user-attachments/assets/8a3d0f9c-18ca-43c2-8b25-132a97381ecc)

这样写可以吗？

**Comment by nbdszw** [comment]
Yes, there are some files missing from this project. This is because the paper it relates to is under review, and we will upload all the code when the paper is officially published.
